recipes/perl/assets/create_assetGroup.pl

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env perl
+
+require 'api_requests.pl';
```

Comment:
This is pulling in a file from a sibling directory. Do we need to modify `@INC` or something for this to work?

Reply:
The PERL @INC was removed to some security reasons. Went Back to code what i have to work the script.